source/guide_coturn.rst

```diff
@@ -92,22 +94,22 @@ Replace values in brackets ``<value>`` with your values.
  listening-ip=0.0.0.0
  relay-ip=::
  relay-ip=0.0.0.0
+ external-ip=<IP_OF_YOUR_UBERSPACE>
  min-port=<port-3>
  max-port=<port-5>
  fingerprint
- lt-cred-mech
  use-auth-secret
  static-auth-secret=<YOUR_SUPER_LONG_SUPER_SECRET_STATIC_PASSPHRASE>
- realm=isabell.uber.space
- total-quota=100
+ realm=<isabell>.uber.space
```

Comment:
This should be `<username>.uber.space`. Please also add a note block above about the needed replacement so users know that they need to do something.

```suggestion
 realm=<username>.uber.space
```